src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -680,8 +680,10 @@ void StridedSlice::StridedSliceCommonExecutor::dimsGluing() {
 
     std::pair<size_t, size_t> secondDim = { 0, params.attrs.begin.size() };
     VectorDims indexes(1, 0);
-    for (int idx = 0; idx < params.attrs.begin.size(); idx++) {
-        if (params.attrs.begin[idx] != 0 || params.attrs.end[idx] != params.srcBlockedDims[idx] - 1 || params.attrs.stride[idx] != 1) {
+    for (int idx = 0; idx < static_cast<int>(params.attrs.begin.size()); idx++) {
```

Comment:
```suggestion
    for (int idx = 0; idx < static_cast<int>(params.attrs.begin.size()); idx++) {
```
```suggestion
    for (size_t idx = 0; idx < params.attrs.begin.size(); idx++) {
```
Mostly used as `size_t`

Reply:
I know your idea, however it is very strange.
`indexes.push_back(std::max(idx - 1, 0));`
`idx-1` maybe negative value, does it work? or current case will not hit `0-1`.
Anyway, I can convert to size_t firstly, because` indexes` also is unsigned.

Reply:
Good point. Thus, we can cast `idx` to a signed type in this check, since only here the sign is needed.

Reply:
Done